crates/bevy_core_pipeline/src/fxaa/mod.rs

```diff
@@ -58,6 +58,16 @@ pub struct Fxaa {
     pub edge_threshold_min: Sensitivity,
 }
 
+impl ExtractComponent for Fxaa {
+    type Query = &'static Self;
+    type Filter = With<Camera>;
+    type Out = (Self, MipBias);
+
+    fn extract_component(item: QueryItem<'_, Self::Query>) -> Option<Self::Out> {
+        Some((item.clone(), MipBias(-0.25)))
```

Comment:
Is there any significance to the `-0.25` value?

Reply:
Copied from Godot https://github.com/godotengine/godot-proposals/issues/4657. I haven't been able to test it out for FXAA though, although I know -1.0 works well for TAA.